taichi/codegen/spirv/kernel_utils.h

```diff
@@ -20,16 +20,11 @@ namespace spirv {
  * Per offloaded task attributes.
  */
 struct TaskAttributes {
-  enum class BufferType {
-    Root,
-    GlobalTmps,
-    Context,
-    ListGen,
-  };
+  enum class BufferType { Root, GlobalTmps, Context, ListGen, ExtArr };
```

Comment:
OOC, do we really support ListGen buffer in SPIR-V codegen..?

Reply:
We have very low level all dense->bitmasked support